huaweicloud/provider.go

```diff
@@ -445,6 +445,8 @@ func Provider() *schema.Provider {
 
 			"huaweicloud_servicestage_component_runtimes": servicestage.DataSourceComponentRuntimes(),
 
+			"huaweicloud_smn_topic": smn.DataSourceTopic(),
```

Comment:
all names of the data source, including functions, files and docs, should be updated to **topics**

Reply:
fixed